src/main/java/duke/Event.java

```diff
@@ -0,0 +1,79 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Event class is a task. The input must be in such a format
+ * "Event <event name> /at <event date>".
+ */
+public class Event extends Task{
+    protected String description;
```

Comment:
I am not sure if having two String description for the children of Task is a good idea since it looks like you are assigning description twice and have 2 description variables. I think you did the same for Event, Deadline and ToDo, I am not sure if this is wise.